### DISCUSSION TITLE
Website hosting

### DISCUSSION THREAD

**Discussion by samajammin** [privacy-ethereum/maci#903] [Ideas]
Looking to open up a discussion on how we might best handle our website hosting for https://maci.pse.dev/ (& any future sites)

In terms of key features, I see only a few main requirements
- support for automated deployments (i.e. whenever we merge website changes into `dev` branch)
- support for preview deploys for PRs (to easily review changes)

Any other requirements I'm missing? What nice-to-haves can folks think of?

Our current setup seems suboptimal:
- GH pages for our production deployment & hosting
- Netlify for PR preview deploys (on a free personal account)

GH pages is free but doesn't have preview deploy functionality? Netlify on a personal account is also free but isn't workable, as there no way for team members to gain access to the account.

Seems a couple options might be:
A) Sign up for a Netlify pro account ($19/month per member) - https://www.netlify.com/pricing/
B) Sign up for a Fleek pro account ($40/month for 3 members) - https://fleek.co/pricing/

As of now, I'd lean towards Fleek. I think it'd be good to use & support the technology projects aligned with our ecosystem, so hosting sites using IPFS infrastructure seems to make sense.

Another consideration - it might be worth looking ahead to what other potential projects we work on (e.g. Coordinator Service, which would be a web app & backend server, not just a static website). Are there future needs we may want to account for now?

**Comment by ctrlc03** [comment]
Agree one solution would be best. 

If we wanted to go with a completely free solution, we could allow GH pages deployments with every PR (coming from within the repo - no forks) - currently we only deploy when the push is to dev - downside of this is that with every PR we could be changing live content and would need to re run the workflow off another branch to fix. Upside is that is free and for it to work we only need to remove one line of code from the workflow + it doesn't send you email notifications if you are subscribed to a PR's updates
